mission/simulator/platform-services-impl/src/main/java/esa/mo/platform/impl/provider/softsim/GPSSoftSimAdapter.java

```diff
@@ -103,4 +103,9 @@ public TwoLineElementSet getTLE()
         tle.getRevolutionNumberAtEpoch());
   }
 
+  public TLE getOrekitTLE()
```

Comment:
Missing `@Override`

Reply:
Also does it make sense to have 2 separate calls on the interface? Note we need to update OPS-SAT Adapter in such case as well.

Reply:
Basically it is possible to get the TwoLineElementSet and convert it back to TLE object, but thought that it would be better to avoid converting it back and forth, when we can get the original object. Though yes, OPS_SAT code needs update as well in this case.

Reply:
Assuming we don't mind breaking the adapter interface, and have to change the OPS SAT mission anyway, is there a more elegant solution?

Reply:
Looking at the way the TwoLineElementSet getTLE() is used (seems that only in this one GPS service class), we could also just change the return type to TLE, and then the conversion to TwoLineElementSet will be done in the service. This way the conversion will only  be done in one place as well - in the service side vs on the adapter implementations. We still need to change the interface, but it will bring the conversion into one class, which I like a lot more.